docs/Feature-Envy.md

```diff
@@ -2,28 +2,48 @@
 
 ## Introduction
 
-Feature Envy occurs when a code fragment references another object more often than it references itself, or when several clients do the same series of manipulations on a particular type of object.
+`Feature Envy` occurs when a code fragment references another object more often than it references itself, or when several clients do the same series of manipulations on a particular type of object.
```

Comment:
I'm not sure about writing smell names in monospace font. Can we offset them some other way?